File: contestOfKotlin/src/test/kotlin/challenges/Challenge1Test.kt
package challenges.challenge1

import org.example.challenges.challenge1.*
import kotlin.test.Test
import kotlin.test.*

class Challenge1TestCase {
    @Test fun testIsNumberMultipleOf() {
        assertTrue { isNumber(1, multipleOf = 1) }
        assertTrue { isNumber(1, multipleOf = 1) }
        assertTrue { isNumber(6, multipleOf = 2) }
        assertTrue { isNumber(20, multipleOf = 5) }
        assertTrue { isNumber(15, multipleOf = 5) }
        assertTrue { isNumber(21, multipleOf = 7) }
        assertTrue { isNumber(72, multipleOf = 4) }
        assertTrue { isNumber(81, multipleOf = 9) }
        assertTrue { isNumber(90, multipleOf = 10) }

        assertFalse { isNumber(3, multipleOf = 2) }
        assertFalse { isNumber(5, multipleOf = 2) }
        assertFalse { isNumber(7, multipleOf = 2) }
        assertFalse { isNumber(21, multipleOf = 5) }
        assertFalse { isNumber(72, multipleOf = 5) }
        assertFalse { isNumber(81, multipleOf = 2) }
        assertFalse { isNumber(90, multipleOf = 8) }
    }
}

File: contest_of_dart/test/challenges/challenge1_test.dart
import 'package:test/test.dart';
import '../../bin/challenges/challenge1.dart';

void main() {
  test('fizz buzz', () {
    // True expectations
    expect(isNumber($: 1, multipleOf: 1), equals(true));
    expect(isNumber($: 2, multipleOf: 2), equals(true));
    expect(isNumber($: 6, multipleOf: 2), equals(true));
    expect(isNumber($: 20, multipleOf: 5), equals(true));
    expect(isNumber($: 15, multipleOf: 5), equals(true));
    expect(isNumber($: 21, multipleOf: 7), equals(true));
    expect(isNumber($: 72, multipleOf: 4), equals(true));
    expect(isNumber($: 81, multipleOf: 9), equals(true));
    expect(isNumber($: 90, multipleOf: 10), equals(true));

    // False expectations
    expect(isNumber($: 3, multipleOf: 2), equals(false));
    expect(isNumber($: 5, multipleOf: 2), equals(false));
    expect(isNumber($: 7, multipleOf: 2), equals(false));
    expect(isNumber($: 21, multipleOf: 5), equals(false));
    expect(isNumber($: 72, multipleOf: 5), equals(false));
    expect(isNumber($: 81, multipleOf: 2), equals(false));
    expect(isNumber($: 90, multipleOf: 8), equals(false));
  });
}
